backend/battling/api/serializers.py

```diff
@@ -20,17 +22,26 @@ class Meta:
 
 class TeamSerializer(serializers.ModelSerializer):
     trainer = UserSerializer()
-    pokemons = PokemonSerializer(many=True)
 
     class Meta:
         model = Team
-        fields = ("id", "battle", "trainer", "pokemons")
+        fields = ("id", "battle", "trainer")
+
+
+class PokemonTeamSerializer(serializers.ModelSerializer):
+    teams = TeamSerializer()
+    pokemon = PokemonSerializer(many=True)
+
+    class Meta:
+        model = PokemonTeam
+        fields = ("id", "teams", "pokemon", "order")
 
 
 class BattleSerializer(serializers.ModelSerializer):
     creator = UserSerializer(read_only=True)
     opponent = UserSerializer(read_only=True)
     teams = TeamSerializer(many=True, read_only=True)
+    pokemon_teams = PokemonTeamSerializer(many=True, required=False)
```

Comment:
I don't see the utility of this attribute, where are you thinking about using this?